x-pack/test/functional/page_objects/security_page.ts

```diff
@@ -488,21 +491,14 @@ export class SecurityPageObject extends FtrService {
       await this.monacoEditor.setCodeEditorValue(roleObj.elasticsearch.indices[0].query);
     }
 
-    const globalPrivileges = (roleObj.kibana as any).global;
```

Comment:
__note:__  I removed the `kibana` argument since in all usages of `addRole` we specify the same value for this argument. If we ever need to customize it in the future, we can easily recover this logic.